### ISSUE TITLE
Latest package not generating type mappings

### ISSUE THREAD

**Issue by digarza** [microsoft/fluentui-system-icons#315]
Latest package version 1.1.138 is not generating any type mapping files for the icons, is this a know issue or am I missing something setup-wise? 

Might be related to this PR: https://github.com/microsoft/fluentui-system-icons/pull/314

**Comment by tomi-msft** [comment]
Hi @digarza, thanks for filing the issue. I removed the mapping files from the exported files in the production build because it doubles the number of exported files, which slows down download speeds for things like codesandbox.

**Comment by digarza** [comment]
thanks for responding. Sorry if this question is not related to the issue but, how could we avoid getting typescript compiler errors when it's not finding the type mappings? Should we declare those ourselves or is there guidance to consume this package with TS?

**Comment by JasonGore** [comment]
@tomi-msft This seems like an optimization that shouldn't affect development consumption of packages. Typings are used by the vast majority of consumers and removing these types is causing all sorts of compiler breaks due to missing type information. If you want to optimize for a niche scenario can you perhaps publish a special package for those use cases instead?

**Comment by JasonGore** [comment]
If `src` was previously published (per #314) then consumers were likely typings from that. It doesn't appear there are any type declaration (`d.ts`) files in [the published output](https://unpkg.com/browse/@fluentui/react-icons@1.1.138/lib/cjs/components/) which means that removing the `src` breaks typing for consumers. Can you generate `d.ts` files in the published output as a replacement for typings lost with the removal of `src`?

**Comment by tomi-msft** [comment]
Hello! I have published a PR which should address this issue. I have readded type mappings to the published output in the `lib` folder. I will be further optimizing the typings files by consolidating them into one in a separate PR to work towards the end goal of reducing the total number of files emitted by the package.

Upgrading to the newest version should readd the d.ts files necessary to prevent these compiler errors!

https://github.com/microsoft/fluentui-system-icons/pull/319
